### PR TITLE
updated README with `cargo component serve` instructions

### PR DESCRIPTION
**Pull request by calvinrp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ So without further ado...
 ## Let's go!
 
 First, [install `cargo component`](https://github.com/bytecodealliance/cargo-component#requirements),
-version 0.11.0, which is a tool for building Wasm components implemented in
+which is a tool for building Wasm components implemented in
```
Comment (sunfishcode) [inline README.md:17]: I sometimes get bug reports from people who have older versions of cargo-component installed. Could you add words like "at least version 0.13" or so here?

File: README.md [hunk 2]
```diff
@@ -40,12 +40,11 @@ $ curl https://wasmtime.dev/install.sh -sSf | bash
...
 [wasmtime.dev]: https://wasmtime.dev/
 
-Then, in a new terminal, we can run `wasmtime serve` on our Wasm component:
+Run `cargo component serve` or the equivalent:
 ```
 $ wasmtime serve target/wasm32-wasi/debug/hello_wasi_http.wasm
 ```
-This starts up an HTTP server on `0.0.0.0:8080` (the specific address and port
-can be configured with the `--addr=` flag).
+This starts up an HTTP server on `0.0.0.0:8080`.
```
Comment (sunfishcode) [inline README.md:47]: How about "This starts up an HTTP server that by default listens on `0.0.0:8080`"?

File: README.md [hunk 3]
```diff
@@ -40,12 +40,11 @@ $ curl https://wasmtime.dev/install.sh -sSf | bash
 
 [wasmtime.dev]: https://wasmtime.dev/
 
-Then, in a new terminal, we can run `wasmtime serve` on our Wasm component:
+Run `cargo component serve` or the equivalent:
 ```
 $ wasmtime serve target/wasm32-wasi/debug/hello_wasi_http.wasm
```
Comment (sunfishcode) [inline README.md:45]: Could you change the example command here to use `cargo component serve`? I want to make it copy-and-pastable.

**Review by sunfishcode — COMMENTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (calvinrp) [reply README.md:17]: Yeah, I was wondering why there was a version number in there. Which gets out of date. I'll add that language.

File: README.md (hunk 2 above)
Comment (calvinrp) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (calvinrp) [reply README.md]: Updated

**Pull request merged**